<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transactions</class>
 <widget class="QDialog" name="Transactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1536</width>
    <height>830</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>0</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">object-fit: cover;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="settings">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>942</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>434</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:none;
border-radius: 8;
font-size: 16px;
font-weight: bold;
padding-right: 50px;
color:white;
}

QPushButton:hover {
	border: 1.5px solid white;
}
</string>
   </property>
   <property name="text">
    <string>Card</string>
   </property>
  </widget>
  <widget class="QLabel" name="account">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>1002</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logOut">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>1002</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border-radius: 8;
font-size: 16px;
padding-right:50px;
color:white;
 font-weight: bold;
}

QPushButton:hover {
	border: 1.5px solid white;
}
</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Card">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>439</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fileImg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>864</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #A3BAA9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>366</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:1.5px solid white;
border-radius: 8;
font-size: 16px;
padding-left:50px;
 font-weight: bold;
color:white;
}

</string>
   </property>
   <property name="text">
    <string>Transaction History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>720</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
font-family: inter;
background: transparent;
border:none;
text-align: right;
color:white;
</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>770</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_username_t">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>765</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:18px;
color:white;
</string>
   </property>
   <property name="text">
    <string>Example Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="logOut_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>770</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>755</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="settings_2">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>719</y>
     <width>41</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>1308</x>
     <y>152</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>142</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Transactions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>676</x>
     <y>143</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>953</x>
     <y>141</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 40px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Category</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 45px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Transaction History</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>317</x>
     <y>60</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>312</x>
     <y>70</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 25px;
font-family: inter;</string>
   </property>
   <property name="text">
    <string>Manage your transactions here if needed!</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1252</x>
     <y>14</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
border-radius: 10;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>39</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;
font-family: inter;
color: #6750A4;
border: none;</string>
    </property>
    <property name="text">
     <string>$111.111.111</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;
font-family: inter;
border:none;</string>
    </property>
    <property name="text">
     <string>Current Balance</string>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>1231</width>
     <height>591</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1229</width>
      <height>589</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="transactionLayout"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>423</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border:none;
border-radius: 8;
font-size: 16px;
font-weight: bold;
padding-right: 50px;
color:white;
}

QPushButton:hover {
	border: 1.5px solid white;
}
</string>
   </property>
   <property name="text">
    <string>Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background: transparent;
border-radius: 8;
font-size: 16px;
padding-right:50px;
color:white;
 font-weight: bold;
}

QPushButton:hover {
	border: 1.5px solid white;
}
</string>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Card_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>439</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fileImg_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>372</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Logo_2">
   <property name="geometry">
    <rect>
     <x>-13</x>
     <y>0</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">object-fit: cover;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>770</y>
     <width>34</width>
     <height>34</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none;
border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
